src/test/java/features/AddRoleFt.java

```diff
@@ -45,7 +44,7 @@ public void an_authorized_client() {
     @When("the authorized client sends a {string} request")
     public void the_authorized_client_sends_a_request(String httpMethod) {
         this.scenarioContext.setRequestBody(new ConcurrentHashMap<>());
-        scenarioContext.getRequestBody().put(HTTP_METHOD, httpMethod.toUpperCase());
+        scenarioContext.getRequestBody().put(HTTP_REQUEST_FIELD_FOR_HTTP_METHOD, httpMethod.toUpperCase());
```

Comment:
Given the nature of these kinds of simple restructuring PRs, I think we should generally avoid adding even simple unrelated changes to the PR (I do this myself and I realise that it is a mistake).